doc/classes/Variant.xml

```diff
@@ -32,7 +38,17 @@
 		        # Note also that there is not yet any way to get a script's `class_name` string easily.
 		        # To fetch that value, you need to dig deeply into a hidden ProjectSettings setting: an Array of Dictionaries called "_global_script_classes".
 		        # Open your project.godot file to see it up close.
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+        int foo = 2;
+        if (foo == null) {
+            GD.Print("foo is null");
+        }
+        if (foo is int) {
+            GD.Print("foo is an integer");
+        }
```

Comment:
The indentation to the left of a codeblock needs to be tabs.

Reply:
fixed